sdk/core/azure-core/src/http/retry_policy.cpp

```diff
@@ -53,6 +53,18 @@ bool GetResponseHeaderBasedDelay(RawResponse const& response, std::chrono::milli
   return false;
 }
 
+/**
+ * @brief Calculate the exponential delay needed for this retry.
+ *
+ * @param retryOptions Options controlling the delay algorithm.
+ * @param attempt Which attempt is this?
+ * @param jitterFactor Test hook removing the randomness from the delay algorithm.
```

Comment:
Is `jitterFactor` just used as a test hook?

Looks like it is relevant for anyone working with a RetryPolicy and if they want to control the randomness:
https://github.com/Azure/azure-sdk-for-cpp/blob/dfe9a2be1f09f932ce0e36ae13c660cbad5f88a5/sdk/core/azure-core/inc/azure/core/http/policies/policy.hpp#L346

Reply:
As currently used, it is a test hook only. We can add it if we get customers who want to change this, but allowing the ability to remove the jitter violates the SDK coding guidelines (which mandate jitter).